### PR TITLE
bugs-fix

### PR DESCRIPTION
**Pull request by varunrampal**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -31,16 +31,11 @@ const App = () => {
     );
     } else {
       routes = (
-        <Switch>
-            <Route path="/" exact>
-                <SigninPage />
+      <Switch>
+            <Route path="/dashboard">
+                <Dashboard />
```
Comment (legenddaniel) [inline client/src/App.js:36]: Is this dashboard with the header?

File: server/controllers/receipt-controller.js [hunk 2]
```diff
@@ -5,13 +5,13 @@ const HttpError = require('../helpers/http-error');
...
 const createReceipt = async (req, res, next) => {
-    const { title, amount, category, date, picture } = req.body;
+    const { title, amount, category, date, picture, user } = req.body;
 
     try {
         //create new receipt
         const newReceipt = new Receipt({
             title: title,
-            user: req.userData.userId,
+            user: user,
```
Comment (singonwan) [inline server/controllers/receipt-controller.js:14]: so now users are going to be passed through the body of the http request? No need to get it from the header from token?

Comment (varunrampal) [reply server/controllers/receipt-controller.js:14]: Good, question, Actually I was having some problem with this route so I was trying different options. We shall get the userid from headers only. I will change it.

File: client/src/App.js (hunk 1 above)
Comment (varunrampal) [reply client/src/App.js:36]: No, you have to put header and side bar

**Pull request merged**